osidb/api_views.py

```diff
@@ -680,6 +681,49 @@ class FlawPackageVersionView(
     permission_classes = [IsAuthenticatedOrReadOnly]
 
 
+def action_at_empty_prefix():
```

Comment:
Hi @JakubFrejlach any idea, please, whether there's a nicer way to generate valid openapi.yml for a bulk PUT handler at `^affects/$` while the usual PUT at `^affects/(?P<pk>[^/.]+)$` is still there? Without this magic and `@action`, drf-spectacular ignores the custom `put` method and only sees the usual single-instance PUT at `^affects/(?P<pk>[^/.]+)$` .

Reply:
@JakubFrejlach I've went through your research and indeed this is a very delicate corner case :smile: Unfortunately I don't think there will be a better solution. The only thing which comes to my mind and would probably worth to consider is to just go with some name for the bulk endpoint and avoid using `action_at_empty_prefix`.

Eg. having:
`PUT /affects/bulk`

and later probably also `DELETE /affects/bulk` and `POST /affects/bulk` to really emphasize that these are the bulk endpoints. I don't see a problem with referencing bulk endpoints with this extra keyword in URL. Wdyt?

Reply:
I agree.